### ISSUE TITLE
May 23 Issue - Project Progress

### ISSUE THREAD

**Issue by ansel-rangers11** [anneguo3/budget-buddy#1]
Describe (for the project): (1) what you’ve worked on in the previous two weeks, (2) what you’ll be working on in the next two weeks, and (3) where you might have gotten stuck during the last two weeks.

**Comment by ansel-rangers11** [comment]
(1) I have supported my group members in brainstorming project scope and description to flesh out what our project is. 
(2) Setting up the Node app and contributing to my team with my previous experience in using Redux to set up a scalable system
(3) Nothing specific

**Comment by elaineay** [comment]
1. Helped collaborate in discussing our minimum, standard, stretch requirements as well as the general outline of our project. Specifically, I created the rough mockups.
2. Help in the creation of components for our app, likely for budgeting items
3. there weren't very many issues in the last 2 weeks since they were very broad discussions on how to begin

**Comment by anneguo3** [comment]
1. Worked alongside my group in determining the scope of the requirements we hope to tackled for the project, set up or repository and updated our README
2. Create components crucial to our minimal requirements, building out the page where individuals can add entries, and collaborate in making sure no one is stepping on anyone else's toes
3. Not many challenges, spent quite a bit of time mulling over options for our project, but worked together in landing on an idea that we all felt happy and comfortable with

**Comment by sylvstrc** [comment]
1. I have worked with the team to brainstorm project ideas, narrow down the scope and help come up with minimal, standard and stretch requirements.
2. Create React components to help fulfill the minimal requirements of the project. Work on the CSS style sheet to adding styles, colors, margins, etc to the front end.
3. The biggest challenge was deciding which project idea to proceed with but after listing out the pros and cons we were able to move forward with a clear decision.
